### PR TITLE
Some updates to the light and dark themes.

### PR DESCRIPTION
**Pull request by flukeout**
Updated the light and dark theme a bit. It's a start. Also made it default to the dark theme.

### CODE REVIEW

File: src/extensions/default/bramble/lib/Theme.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ define(function (require, exports, module) {
 
     function toggle(data) {
         var theme = data.theme || currentTheme;
-        setTheme(theme === "light-theme" ? "dark-theme" : "light-theme");
+        setTheme(theme === "light-theme" ? "dark-theme" : "dark-theme");
```
Comment (humphd) [inline src/extensions/default/bramble/lib/Theme.js]: This change is wrong, can you revert?

File: src/extensions/default/bramble/lib/Theme.js [hunk 2]
```diff
@@ -57,7 +57,7 @@ define(function (require, exports, module) {
         ThemeManager.addTheme(lightFile, lightOptions);
         ThemeManager.addTheme(darkFile, darkOptions);
 
-        currentTheme = theme === 'light-theme' ? "dark-theme" : "light-theme";
+        currentTheme = theme === 'light-theme' ? "dark-theme" : "dark-theme";
```
Comment (humphd) [inline src/extensions/default/bramble/lib/Theme.js]: I'm not sure what this code was meant to do, but it's also wrong.  We should just take `theme` as passed and use that, or stick with whatever `currentTheme` is.  I'd say this code got messed up in the recent past, which might explain why it doesn't retain the choice with a refresh.

File: src/extensions/default/bramble/stylesheets/darkTheme.less [hunk 3]
```diff
@@ -58,10 +70,20 @@
  * @TODO Scrollbars
  * @TODO Find highlights
  */
+
+.CodeMirror-gutter:after {
+  content:"";
+  position: absolute;
+  height: 100%;
+  width: 0px;
+  right: 0px;
```
Comment (humphd) [inline src/extensions/default/bramble/stylesheets/darkTheme.less]: Drop `px` on `0` values.

File: src/extensions/default/bramble/stylesheets/lightTheme.less [hunk 4]
```diff
@@ -21,7 +21,7 @@ span.cm-tag			{ color: #002cb8; }
 span.cm-attribute	{ color: #008d5c; }
 span.cm-property	{ color: #008d5c; } /* property after string */
 
-span.cm-comment		{ color: #779cac; }
+span.cm-comment		{ color: #B1B1B1; }
 span.cm-error		{ color: #ff3da5; }
 span.cm-keyword		{ color: #002ea3; }
 span.cm-variable	{ color: #0056ba; }
```
Comment (humphd) [inline src/extensions/default/bramble/stylesheets/lightTheme.less]: Odd indents going on in the previous block, can you normalize for 4-space indent?

File: src/extensions/default/bramble/stylesheets/lightTheme.less [hunk 5]
```diff
@@ -29,9 +29,22 @@ span.cm-variable	{ color: #0056ba; }
...
+.CodeMirror-linenumber {
+  color: #CCC;
+}
+
+.CodeMirror-gutter:after {
+  content:"";
+  position: absolute;
+  height: 100%;
+  width: 0px;
+  right: 0px;
```
Comment (humphd) [inline src/extensions/default/bramble/stylesheets/lightTheme.less]: No `px on `0`.

File: src/styles/brackets.less [hunk 6]
```diff
@@ -521,9 +524,9 @@ a, img {
 
 // Split View Separator Styles
 .split-vertical #second-pane {
-    border-left: 1px solid rgba(0, 0, 0, 0.17);
+    // border-left: 1px solid rgba(0, 0, 0, 0.17);
```
Comment (humphd) [inline src/styles/brackets.less]: Let's remove vs. commenting-out.

**Comment by flukeout** [comment]
@humphd Fixed the problems you pointed out - I left the "default theme" and "remembered theme" logic as it was - and we can sort that out via separate issues.

* https://github.com/mozilla/thimble.webmaker.org/issues/905
* https://github.com/mozilla/thimble.webmaker.org/issues/904

**Comment by humphd** [comment]
Looks good.  One thing before I can land this, I need you to revert your changes to `src/styles/brackets.less`, otherwise rebasing on Adobe's stuff later is going to be a pain.  If you really need to make changes at that level, do it in `src/styles/bramble_overrides.css`.

Once you do that, r=me.

**Pull request closed without merging**